com.stansassets.scene-management/BuildConfigurator/Editor/BuildConfigurationWindow.cs

```diff
@@ -144,7 +144,16 @@ void DrawConfiguration(int index)
             using (new IMGUIBlockWithIndent(new GUIContent("Settings")))
             {
                 conf.Name = IMGUILayout.TextField("Configuration Name:", conf.Name);
+                
+                EditorGUI.BeginChangeCheck();
                 conf.DefaultScenesFirst = IMGUILayout.ToggleFiled("Default Scenes First", conf.DefaultScenesFirst, IMGUIToggleStyle.ToggleType.YesNo);
+                if (EditorGUI.EndChangeCheck())
+                {
+                    if (m_AutoSyncParams.Synced)
+                    {
+                        SyncScenes();
+                    }
```

Comment:
Why aren't we using here 'CheckNTryAutoSync' ?

Reply:
If m_AutoSyncParams.Synced == true then we need to "force" sync without conditional validation that that are presented in 'CheckNTryAutoSync' method. Otherwise, configuration and build settings will be out of sync and we will loss the "synced" status of our actions. The reason is that the second configuration should be applied as if it were the default.